src/_base/_twig/docker-compose.yml/service/webapp.yml.twig

```diff
@@ -1,3 +1,2 @@
 {% set blocks  = '_twig/docker-compose.yml/' %}
 {% include blocks ~ 'service/nginx.yml.twig' %}
-{% include blocks ~ 'service/php-fpm.yml.twig' %}
```

Comment:
I think its a bit confusing only having nginx here, and hence a similar problem if we add enabled to nginx.